### PR TITLE
Update gimp.json

### PR DESCRIPTION
**Pull request by jophab**
updated name,source name,url,desc

### CODE REVIEW

File: share/goodie/cheat_sheets/json/gimp.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
     "id": "gimp_cheat_sheet", 
-    "name": "Gimp",
-    "description": "Graphics Editor", 
+    "name": "GIMP",
+    "description": "Keyboard shortcuts for GIMP(GNU Image Manipulation Program) graphics editor",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/gimp.json]: Nit: Should be `GIMP (Gnu Image Manipulation Program`

Comment (jophab) [inline share/goodie/cheat_sheets/json/gimp.json]: @mintsoft  are you telling about the name or the desc?

Comment (mintsoft) [inline share/goodie/cheat_sheets/json/gimp.json]: @jophab The description, it's missing a space before the `(`

Comment (jophab) [inline share/goodie/cheat_sheets/json/gimp.json]: :+1:  I will do it

File: share/goodie/cheat_sheets/json/gimp.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
 {
     "id": "gimp_cheat_sheet", 
-    "name": "Gimp",
-    "description": "Graphics Editor", 
+    "name": "GIMP",
+    "description": "Keyboard shortcuts for GIMP (GNU Image Manipulation Program) graphics editor", 
     "metadata": { 
-        "sourceName": "GimpCheatSheet"
+        "sourceName": "Jeffkayser",
+        "sourceUrl": "https://jeffkayser.com/static/cheatsheets/cheatsheet_gimp-letter.pdf"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gimp.json]: I couldn't get this file to render in my browser or on my Mac -- Are there any other sources?

Comment (moollaza) [inline share/goodie/cheat_sheets/json/gimp.json]: This seems to work better: https://jeffkayser.com/static/cheatsheets/cheatsheet_gimp.png

**Comment by jagtalon** [comment]
:+1: nice!

**Pull request merged**